v8.0/src/app/components/photo-editor.component.ts

```diff
@@ -0,0 +1,48 @@
+import { Component, OnInit, ViewChild, Input } from "@angular/core";
+// @ts-ignore
+import { PhotoEditorSDKUI } from "photoeditorsdk";
+
+/* React Magic */
+import * as React from "react";
+import * as ReactDom from "react-dom";
+
+declare global {
+  interface Window {
+    React: any;
+    ReactDom: any;
+  }
+}
+
+window.React = window.React || React;
+window.ReactDom = window.ReactDom || ReactDom;
+
+const license = "";
+
+@Component({
+  selector: "app-photo-editor",
+  templateUrl: "./photo-editor.component.html"
+})
+export class PhotoEditorComponent implements OnInit {
+  constructor() {}
+
+  @Input() src: string;
+  @ViewChild("psdkContainer", { static: false }) container;
+
+  image = new Image();
+  editor;
+
+  ngOnInit() {
+    this.image.crossOrigin = "anonymous";
+    this.image.onload = this.instantiateEditor.bind(this);
+    this.image.src = this.src;
```

Comment:
It would be also great, if the demo image does not rely on `onload` of the image event, `PhotoEditorSDK UI ` is capable of doing that internally  and it makes the getting started easier